swarm/pss/pss.go

```diff
@@ -179,40 +213,82 @@ func (self *Pss) Register(topic *Topic, handler Handler) func() {
 	return func() { self.deregister(topic, &handler) }
 }
 
-func (self *Pss) deregister(topic *Topic, h *Handler) {
+// Set the amount of bytes that will be disclosed encrypted of recipient address
+//
+// 0 will equal whisper routing (forward to all, no unencrypted partial address)
+// -1 will return to default value
+func (self *Pss) SetRecipientAddressLength(l int) {
```

Comment:
I would argue that the security liability just as well is by default NOT to truncate.